packages/utils/docs/report.md

```diff
@@ -0,0 +1,93 @@
+# Code Pushup Report
+
+<!-- START: Meta Section -->
+
+---
+
+_Version: 0.0.1_  
+_Date: 9/11/2023 12:00:00am (430ms)_  
+_Plugins: 3; Audits: 44_
+
+---
+
+<!-- STOP: Meta Section -->
+
+<!-- START: Overview Section -->
+<!-- Only audits with weight > 0 are listed in the overview section -->
+
+<!-- Overview Version 1 -->
+
+| Performance | A11y | NxAdoption |
+| ----------- | ---- | ---------- |
+| 60          | 34   | 80         |
+
+<!-- Overview Version 2 -->
+
+| Category    | Score | Audits |
+| ----------- | ----- | ------ |
+| Performance | 60    | (3/4)  |
+| A11y        | 34    | (7/24) |
+| NxAdoption  | 80    | (7/12) |
```

Comment:
I do not get how you can concatinate the NxAdo... report and the LH report. I say this because of the multiple steps and flow. 

It seems like we will need to generate a different one for each Auditor/Plugin (eslint, nx-valiator, user-flow). 

Additionally the question we should be asking is about headers, column and row headers. 

NxApdoption will probably end up worth and Categories too. 

As we got essentially unlimited vertical space then I guess we need to go in this direction, 

I also would strongly suggest we do what we did with user-flow where we just reduce the report and convert that into a table.